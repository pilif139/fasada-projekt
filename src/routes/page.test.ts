import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/svelte';
import Page from './+page.svelte';

describe("Main Page", ()=>{
  it('should render', () => {
    const {container} = render(Page);
    expect(container).toMatchSnapshot();
  });
})