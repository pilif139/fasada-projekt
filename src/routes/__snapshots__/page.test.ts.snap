// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Main Page > should render 1`] = `
<div>
  <main
    class="flex flex-col items-center text-xl gap-8 w-[40vw] text-center"
  >
    <div
      class="text-6xl text-blue-500 font-nunito font-extrabold"
    >
      Fasada
    </div>
     
    <p
      class="text-2xl"
    >
      Dzięki tej stronie dowiesz się czym jest wzorzec projektowy fasady
    </p>
     
    <a
      class="bg-blue-500 px-8 py-4 rounded-xl text-xl font-bold hover:bg-blue-800 transition-all border-2 border-blue-600 hover:scale-110 duration-300 "
      href="content/0"
    >
      Next
    </a>
    
  </main>
  
\`
</div>
`;
